Shared/Samples/Show utility associations/ShowUtilityAssociationsView.swift

```diff
@@ -0,0 +1,268 @@
+            map.addUtilityNetwork(network)
+        }
+        
+        deinit {
+            ArcGISEnvironment.authenticationManager.arcGISCredentialStore.removeAll()
+        }
+        
+        // MARK: Methods
+        
+        /// Performs important tasks including adding credentials, loading and adding operational layers.
+        func setup() async throws {
+            try await ArcGISEnvironment.authenticationManager.arcGISCredentialStore.add(.publicSample)
+            try await network.load()
+            addLayers()
+        }
+        
+        private func addLayers() {
+            // Get all the edges and junctions in the network.
+            guard let networkSources = network.definition?.networkSources else { return }
+            let sourcesByType = Dictionary(grouping: networkSources, by: \.kind)
+            
+            // Add all edges that are not subnet lines to the map.
+            let edgeLayers = sourcesByType[.edge, default: []]
+                .filter { $0.usageKind != .subnetLine }
+                .map { FeatureLayer(featureTable: $0.featureTable) }
+            
+            map.addOperationalLayers(edgeLayers)
+            
+            // Add all the junctions to the map.
+            let junctionLayers = sourcesByType[.junction, default: []]
+                .map { FeatureLayer(featureTable: $0.featureTable) }
+            map.addOperationalLayers(junctionLayers)
+        }
+        
+        static func makeAssociationsOverlay() -> GraphicsOverlay {
+            let attachmentValue = UniqueValue(
+                description: "Attachment",
+                label: "",
+                symbol: .attachment,
+                values: [UtilityAssociation.Kind.attachment]
+            )
+            let connectivityValue = UniqueValue(
+                description: "Connectivity",
+                label: "",
+                symbol: .connectivity,
+                values: [UtilityAssociation.Kind.connectivity]
+            )
+            
+            let renderer = UniqueValueRenderer(
+                fieldNames: ["AssociationType"],
+                uniqueValues: [attachmentValue, connectivityValue],
+                defaultLabel: ""
+            )
+            
+            let overlay = GraphicsOverlay()
+            overlay.renderer = renderer
+            return overlay
+        }
+        
+        func addAssociationGraphics(viewpoint: Viewpoint, scale: Double) async throws {
+            // Check if the current viewpoint is outside of the max scale.
+            guard isAuthenticated, scale <= maxScale, !isAddingGraphics else { return }
+            isAddingGraphics = true
+            let extent = viewpoint.targetGeometry.extent
+            // Get all of the associations in extent of the viewpoint.
+            let associations = try await network.associations(forExtent: extent)
+            let existingAssociationIDs = Set(associationsOverlay.graphics.map { $0.attributes["GlobalId"] as? UUID } )
```

Comment:
There is a linter warning 

> Closing Brace Spacing Violation: Closing brace with closing parenthesis should not have any whitespaces in the middle (closing_brace)

```suggestion
            let existingAssociationIDs = Set(associationsOverlay.graphics.map { $0.attributes["GlobalId"] as? UUID })
```

Reply:
Fixed, but I don't see any linter warnings in the Samples project. Is there a way to enable linter warnings?